src/display/lv_display.c

```diff
@@ -933,6 +933,36 @@ lv_draw_buf_t * lv_display_get_buf_active(lv_display_t * disp)
     return disp->buf_act;
 }
 
+bool lv_display_get_dirty_area(lv_display_t * disp, lv_area_t * dirty_area)
```

Comment:
How about `lv_display_get_invalid_area` ?

Reply:
move to lv_nuttx